project/Dependencies.scala

```diff
@@ -30,6 +30,7 @@ object Dependencies {
     val lruMap          = "com.snowplowanalytics" %% "scala-lru-map"        % V.lruMap
     // Tests
     val specs2          = "org.specs2"            %% "specs2-core"          % V.specs2 % "test"
+    val specs2Scalaz    = "org.specs2"            %% "specs2-scalaz"        % V.specs2 % "test"
```

Comment:
why is specs2-scalaz required?

Reply:
With spec2 4.0.0 they move `DisjunctionMatchers` to that module.
https://github.com/etorreborre/specs2/releases/tag/SPECS2-4.0.0

It seems the trait is there (`CacheSpec` and `DarkSkyCacheSpec`) but not used. So the trait and the dependency could be removed. I will push a followup commit to this PR.

Reply:
Cool, could you log the corresponding issue and format your commits like:
- "Bump specs2 to 4.3.4 (closes #91)"
- "Remove DisjunctionMatchers (closes #xx)"

Reply:
are two commits in a single PR ok, or should I make 2 PRs out of this ?

Reply:
2 commits in this PR are fine, I'll rebase and merge :+1: